data/virtualization/autoyast/host_15.xml.ep

```diff
@@ -0,0 +1,305 @@
+<?xml version="1.0"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <suse_register>
+    <do_registration config:type="boolean">true</do_registration>
+    <email><%= $get_var->('SCC_EMAIL') %></email>
+    <reg_code><%= $get_var->('SCC_REGCODE') %></reg_code>
+    <install_updates config:type="boolean">true</install_updates>
+    % if (keys %$addons) {
+    <addons config:type="list">
+      % while (my ($key, $addon) = each (%$addons)) {
+      <addon>
+        <name><%= $addon->{name} %></name>
+        <version><%= $addon->{version} %></version>
+        <arch><%= $addon->{arch} %></arch>
+        % if ($key eq 'we' and $check_var->('SLE_PRODUCT', 'sles')) {
+        <reg_code><%= $get_var->('SCC_REGCODE_WE') %></reg_code>
+        % }
+        % if ($key eq 'we' and $check_var->('SLE_PRODUCT', 'sled')) {
+        <reg_code><%= $get_var->('SCC_REGCODE') %></reg_code>
+        % }
+        % if ($key eq 'rt') {
+        <reg_code><%= $get_var->('SCC_REGCODE_RT') %></reg_code>
+        % }
+        % if ($key eq 'ltss') {
+        <reg_code><%= $get_var->('SCC_REGCODE_LTSS') %></reg_code>
+        % }
+      </addon>
+      % }
+    </addons>
+    %}
+  </suse_register>
+  <add-on>
+    <add_on_products config:type="list">
+      % for my $repo (@$repos) {
+      <listentry>
+        <media_url><%= $repo %></media_url>
+      </listentry>
+      % }
+    </add_on_products>
+  </add-on>
+  <bootloader>
+    <global>
+      <cpu_mitigations>auto</cpu_mitigations>
+      <gfxmode>auto</gfxmode>
+      <hiddenmenu>false</hiddenmenu>
+      <os_prober>false</os_prober>
+      <serial>serial --unit=1 --speed=115200 --parity=no</serial>
+      <timeout config:type="integer">10</timeout>
+      <trusted_grub>false</trusted_grub>
+      <xen_append>splash=silent quiet console=ttyS1,115200</xen_append>
+      % if ($check_var->('VERSION', '15')) {
+      <terminal>serial</terminal>
+      <xen_kernel_append>dom0_max_vcpus=4 dom0_mem=8192M,max:8192M vga=gfx-1024x768x16 loglvl=all guest_loglvl=all</xen_kernel_append>
+      % } else {
```

Comment:
What version is the "else branch" supposed to fit for? version>15?
I assume that host_12.xml.ep is for sle12spx products and host_15.xml.ep is for sle15spx products, right?

Reply:
There is  a lot of  syntax change between autoyast for sle12 and sle15 so we have to create two separate templates.

Reply:
It makes sense to have two templates for sle12 and 15. Would you please answer the above question "What version is the "else branch" supposed to fit for? version>15?"?

Reply:
yes, for all versions >15

Reply:
Interesting... how is it known exactly the setting for something like sle16, which is not available yet?

Reply:
The logic here means:
Set grub2 to use "serial" and "console" in terminal tag for all of sle versions but 15GA.  15GA does support using both of them. 
This autoyast template is just for sle15. sle16 isn't covered.

Reply:
Aha, okay. $check_var->('VERSION', '15') stands for 15GA.